lib/udevnotifier.cpp

```diff
@@ -88,20 +88,33 @@ void UdevNotifier::run()
             // XXX
 //             qDebug() << "hotplug[" << udev_device_get_action(dev) << "] " << udev_device_get_devnode(dev) << "," << udev_device_get_subsystem(dev) << "," << udev_device_get_devtype(dev);
             // emit the found device
-            Q_EMIT udevEvent(actionFromString(udev_device_get_action(dev)), new Device(dev));
+            QString det= udev_device_get_devtype(dev);
+         //   qDebug()<<det;
+            QString det1= "usb_device";
```

Comment:
no need to create seperate QString here. You can do the check in the if condition directly